SETUP.md

```diff
@@ -36,6 +36,19 @@
       gcloud app deploy --project ${BBQ_PROJECT_ID} app.yaml config/cron.yaml config/queue.yaml config/index.yaml
       ```
       Note: If it is your first App Engine deploy, App Engine needs to be initialised and you will need to choose [region/location](https://cloud.google.com/appengine/docs/locations). It is recommended to pick the same location as where most of your BigQuery data resides.
+1.   Secure your application by following given steps.
+     * Restrict IAM roles for your GAE service account and Enable firewall for GAE application.
+       * GAE Editor needs to be removed manually, as it is set by default.
+         ```bash
+         gcloud projects remove-iam-policy-binding ${BBQ_PROJECT_ID} --member='serviceAccount:'${BBQ_PROJECT_ID}'@appspot.gserviceaccount.com' --role='roles/editor’
+         ```
+       * You should follow PoLP (Principle of least privilege) during process of set up IAM's.
+         You can do that via [Terraform](TERRAFORM_SETUP.md).
+         After setup go to **terraform/bbq** directory and run following command:
+         ```bash
+         terraform apply
+         ```
+     * Turn on [IAP](https://cloud.google.com/iap/docs/app-engine-quickstart) (Identity-Aware Proxy) for your GAE application.       
 
```

Comment:
I would add a step to whitelist current user IP in GAE firewall with gcloud command as well